### PR TITLE
docs(ecosystem): add mediumFastify plugin

### PR DESCRIPTION
**Pull request by RjManhas**
added mediumFastifyPlugin to community plugins

### CODE REVIEW

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -663,6 +663,7 @@ middlewares into Fastify plugins
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+- [`mediumFastifyPlugin`](https://github.com/devoplx/mediumFastifyPlugin) A simple plugin for using medium.com api.
```
Comment (climba03003) [inline docs/Guides/Ecosystem.md:666]: Can you place in alphabetically order?

**Review by climba03003 — CHANGES_REQUESTED** [review]

**Comment by Uzlopak** [comment]
I dont know how to say that diplomatically.

Usually if I see a new plugin for the ecosystem, I check the npm package content to check if there is some malware to avoid promoting malicious packages on the fastify webpage. But I am kind of disappointed that this package is basically overbloated and not at all optimized for performance. 

@mcollina 
I think we should actually think about some quality policies for the plugins we are promoting on our page.

**Comment by mcollina** [comment]
There are a few things that are missing:

* API documentation
* remove unneeded dependencies (fastify, tap, tsup, etc can all go into devdeps)
* move away from a [singleton](https://github.com/devoplx/mediumFastifyPlugin/blob/07d50a177fc9142aeefb9750e1934a2339459832/src/services/api.ts#L139) and move initialization inside the [plugin constructor](https://github.com/devoplx/mediumFastifyPlugin/blob/07d50a177fc9142aeefb9750e1934a2339459832/src/index.ts#L19)

**Comment by Uzlopak** [comment]
Closing due to inactivity

**Pull request closed without merging**